src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,42 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
The access modifier for the `getStatistic` method should be carefully considered. If it is not intended to be used outside of this class, it would be better to reduce its visibility to package-private or private.